Assets/MixedRealityToolkit.SDK/Experimental/NonNativeKeyboard/Scripts/SliderInputField.cs

```diff
@@ -12,8 +12,10 @@ namespace Microsoft.MixedReality.Toolkit.Experimental.UI
     /// </summary>
     public class SliderInputField : TMP_InputField
     {
+#pragma warning disable 0169
```

Comment:
Can you add comment of what 0169 is and why we disable it?

Reply:
Turns out it looks like this code wasn't rendering in the inspector anyway.
I instead made a custom inspector with the experimental feature readout

Reply:
![image](https://user-images.githubusercontent.com/3580640/70740244-98768900-1ccd-11ea-93f7-eb8ada4b0802.png)